### ISSUE TITLE
Weird interaction between SAS, LOCK Steering, and connection manager

### ISSUE THREAD

**Issue by Wercho** [KSP-KOS/KOS#3025]
Ok, this is an odd one. There is some sort of weird interaction when SAS is turned off by a program when you don't have a control connection back to home. (eg. "homeconnection:isconnected" is false). Turning SAS off while not connected results in LOCK steering doing nothing. The SAS button on the navball turns off, but the craft does not respond. 

ALSO, the player is able to input steering controls despite no connection. Additionally, there is some SAS-like action that cancels any angular momentum caused by player input. To me, it appears that SAS is still active despite the navball button being dark . 

To test, I created an example craft with a probe core with reaction wheels and antenna, kos processor, fuel tank, engine. (The fuel tank and engine probably aren't necessary.) Then I alt-F12'd it into an orbit that would take it out of comms range and ran the following tests.

With a connection:
- all input functions normally (just to be sure it wasn't something else)

Running the following:
```SAS ON.
LOCK STEERING TO ship:velocity:orbit.
WAIT UNTIL false.
```
SAS and lock steering fight (as expected). No player input accepted. Again, to test what should happen.

This is the main one. Turning sas on with a connection, then off once there is no connection:
```sas on.
wait 1.
wait until NOT(homeconnection:isconnected).
set kuniverse:timewarp:warp TO 0.
wait 1.
sas off.
lock steering to ship:velocity:orbit.
wait until false.
```
When it looses connection, it stops warp and the SAS button goes off, but no steering happens. It just sits there. But if I input controls, the controls register on the thingy in the bottom left corner that shows pitch, yaw, roll inputs. Manual throttle does not have any effect. Inputting a pitch up will cause the ship to rotate up, and then if I stop inputting pitch up a pitch down is automatticaly input to cancel the pitching motion.

Other tests:
If I do "SAS off" before losing connection, then the LOCK steering command works fine after losing connection.
If I do "SAS off" and wait before doing the Lock steering, nothing changes.
If I wait for losing a connection (without SAS) and then do "SAS on" and lock steering, then SAS and lock steering fight like normal.

I'm not sure if this is related in some way to the SAS API changes mentioned in #2117 .

**Comment by nuggreat** [comment]
I half recall something like this coming up before and didn't work out a code based solution.  I think the underlying cause is related to the "allow control without signal" advanced difficulty option changing this setting from what you have it might serve as a work around to this issue.

**Comment by Wercho** [comment]
Yeah, I was testing with "Require Signal for Control" to true. My workaround is just to make sure I always turn SAS off if I don't actually need it, so that my probes don't have it on when they lose connection.
